src/home/homework-luks.c

```diff
@@ -1603,7 +1603,7 @@ int home_activate_luks(
 }
 
 int home_deactivate_luks(UserRecord *h, HomeSetup *setup) {
-        bool we_detached;
+        bool we_detached = false;
```

Comment:
The issue is CID#1469711 should be false positive, as when `acquire_open_luks_device()` returns `1`, `setup->crypt_device` is set.

But the change looks good to me, as it is simpler and clear for readers.